backend/src/components/auth.js

```diff
@@ -36,24 +49,31 @@ const auth = {
 
     try {
       const discovery = await utils.getOidcDiscovery();
-      const response = await axios.post(discovery.token_endpoint,
+      const response = await axios.post(
+        discovery.token_endpoint,
         qs.stringify({
           client_id: config.get('oidc:clientId'),
           client_secret: config.get('oidc:clientSecret'),
           grant_type: 'refresh_token',
           refresh_token: refreshToken,
-          scope: discovery.scopes_supported
-        }), {
+          scope: discovery.scopes_supported,
+        }),
+        {
           headers: {
             Accept: 'application/json',
             'Cache-Control': 'no-cache',
             'Content-Type': 'application/x-www-form-urlencoded',
-          }
+          },
         }
       );
 
       log.verbose('renew', utils.prettyStringify(response.data));
-      if (response && response.data && response.data.access_token && response.data.refresh_token) {
+      if (
+        response &&
+        response.data &&
+        response.data.access_token &&
+        response.data.refresh_token
+      ) {
```

Comment:
you can simpliify with the ? mark null check

' response?.data?.access_token &&
        response?.data?.refresh_token'